### DISCUSSION TITLE
Not recognizing digits on dark backgrounds?

### DISCUSSION THREAD

**Discussion by micahp2** [jomjol/AI-on-the-edge-device#3086] [Q&A]
I have an issue that has happened across multiple versions and installs of this software: white numbers over black backgrounds don't seem to be recognized. Here's a recent example:

![alg_roi](https://github.com/jomjol/AI-on-the-edge-device/assets/3095168/6300d55a-654c-40c8-aa53-741af6c173a2)
The recognition comes back as this: 0064NN0

Perhaps this is some kind of configuration that I am missing or didn't understand. Did anyone else have this issue and if so, how did you solve it?

**Comment by caco3** [comment]
Your ROIs are not setup properly! Please check the documentation!

Looking on the training data on https://jomjol.github.io/neural-network-digital-counter-readout/ the dark digits should be ok.

**Reply by micahp2** [reply]
I will investigate the documentation again and redo the ROIs. Thanks for the info.
